src/server/service/kv.rs

```diff
@@ -976,103 +812,6 @@ impl<T: RaftStoreRouter<E::Local> + 'static, E: Engine, L: LockManager, F: KvFor
         ctx.spawn(task);
     }
 
-    fn read_index(
```

Comment:
Ah, wait, won't this be called by other TiKV nodes?

Reply:
It's also explained in the PR description. It's not used as far as I know.